arthur_bench/models/models.py

```diff
@@ -268,6 +291,11 @@ class HistogramItem(BaseModel):
     high: float
 
 
+class CategoricalHistogramItem(BaseModel):
+    count: int
+    category: float
```

Comment:
Is there a better definition for a category other than float? Maybe some class that can be named / have a description / have a threshold value associated with it?